src/server/tiering/op_manager.cc

```diff
@@ -82,6 +82,7 @@ std::error_code OpManager::Stash(EntryId id_ref, std::string_view value) {
     ProcessStashed(Borrowed(id), version, segment, ec);
   };
 
+  // May block due to blocking call to Grow.
```

Comment:
now that we mentioned it again, is heartbeat (i.e offloading) allowed to block 🤔

Reply:
hearbeat runs in a fiber so technically is allowed.